babel.config.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+  plugins: [
+    "babel-plugin-annotate-pure-calls",
+    "babel-plugin-dev-expression",
+    ["@babel/plugin-proposal-class-properties", { loose: true }],
+    "@babel/plugin-proposal-optional-chaining",
+    "@babel/plugin-proposal-nullish-coalescing-operator",
```

Comment:
those two are included by default now in `@babel/preset-env` as far as I remember - since they have been included in the language spec during Babel 7's lifetime